file-formats/sito/type/zif_aff_sito_v2.intf.abap

```diff
@@ -280,16 +314,16 @@ INTERFACE zif_aff_sito_v1
     ty_sit2_obj_vh_s_p_list TYPE STANDARD TABLE OF ty_sit2_obj_vh_s_p WITH DEFAULT KEY,
 
     "! <p class="shorttext">Text</p>
-    "! Text for ituation object
+    "! Text for situation object
     BEGIN OF ty_sit2_object_t,
       "! <p class="shorttext">Name</p>
       "! Situation object name
       "! $required
-      name        TYPE c LENGTH 40,
-      "! <p class="shorttext">Description</p>
-      "! Description
+      name            TYPE c LENGTH 40,
+      "! <p class="shorttext">Long Description</p>
+      "! Long description
       "! $required
-      description TYPE c LENGTH 255,
+      longdescription TYPE c LENGTH 255,
```

Comment:
Do you want to rename this? I saw (in the old example) there are other "description" fields in text sections.

See: https://github.com/SAP/abap-file-formats/blob/23986fa71df0115e9d7fe9325ffe7bc2de986211/file-formats/sito/examples/zmo_sito_test.sito.json#L25